### PR TITLE
CCP-82 Scroller for description fields

### PR DESCRIPTION
**Pull request by mkarthik90**

### CODE REVIEW

File: Conceptpower+Spring/src/main/webapp/WEB-INF/views/auth/concepts/ConceptListConceptsView.jsp [hunk 1]
```diff
@@ -1,175 +1,180 @@
...
+                    <td align="justify"><font size="2"><c:out
+                                value="${concept.entry.id}"></c:out></font></td>
+                    <td align="justify"><font size="2"><c:out
+                                value="${concept.entry.wordnetId}"></c:out></font></td>
+                    <td align="justify"><font size="2"><c:out
+                                value="${concept.entry.pos}"></c:out></font></td>
+                    <td align="justify"><font size="2"><c:out
+                                value="${concept.entry.conceptList}"></c:out></font></td>
+                    <td align="justify">
+                        <div class="scrollable"
```
Comment (mkarthik90) [inline Conceptpower+Spring/src/main/webapp/WEB-INF/views/auth/concepts/ConceptListConceptsView.jsp]: Changes made to this td. Added a scrollable div for description field

File: Conceptpower+Spring/src/main/webapp/WEB-INF/views/auth/concepts/ConceptWrapperAddView.jsp [hunk 2]
```diff
@@ -1,608 +1,554 @@
...
+                    <form:options items="${lists}" />
+                </form:select></td>
+            <td />
+
+        </tr>
+        <tr>
+            <td>Description</td>
+            <td>
+                <div>
+                    <textarea id="description" cols="60"
```
Comment (mkarthik90) [inline Conceptpower+Spring/src/main/webapp/WEB-INF/views/auth/concepts/ConceptWrapperAddView.jsp]: Changed text field to text area for description.

File: Conceptpower+Spring/src/main/webapp/WEB-INF/views/auth/concepts/ConceptWrapperAddView.jsp [hunk 3]
```diff
@@ -1,608 +1,554 @@
...
+    method='post'>
+    <h2>2. Select Wordnet concept from search results</h2>
+
+    <h4>The following concepts were found:</h4>
+    Remember, you can only create a wrapper for concepts in Wordnet!
+    <p></p>
+
+    <c:if test="${not empty result}">
+        <table cellpadding="0" cellspacing="0"
+            class="table table-striped table-bordered"
```
Comment (mkarthik90) [inline Conceptpower+Spring/src/main/webapp/WEB-INF/views/auth/concepts/ConceptWrapperAddView.jsp]: Removed table from the class attribute. Table has been removed to make screen look better when text area is added.

**Comment by mkarthik90** [comment]
@jdamerow Formatted two files which did not use space indentation. Commented the changes in those files.

**Comment by jdamerow** [comment]
ok to test

**Pull request merged**